versions/3.1.0.md

```diff
@@ -2269,23 +2268,23 @@ $ref: definitions.yaml#/Pet
 
 #### <a name="schemaObject"></a>Schema Object
 
-The Schema Object allows the definition of input and output data types.
-These types can be objects, but also primitives and arrays.
-This object is an extended subset of the [JSON Schema Specification Wright Draft 00](https://json-schema.org/).
+The Schema Object allows the definition of input and output data types. These types can be objects, but also primitives and arrays. This object is a superset of the [JSON Schema Draft 2019-09](http://json-schema.org/) specification.
+
+For more information about the properties, see [JSON Schema Core](https://json-schema.org/draft/2019-09/json-schema-core.html) and [JSON Schema Validation](https://json-schema.org/draft/2019-09/json-schema-validation.html).
 
-For more information about the properties, see [JSON Schema Core](https://tools.ietf.org/html/draft-wright-json-schema-00) and [JSON Schema Validation](https://tools.ietf.org/html/draft-wright-json-schema-validation-00).
 Unless stated otherwise, the property definitions follow the JSON Schema.
 
-##### Properties 
+##### Properties
 
-The following properties are taken directly from the JSON Schema definition and follow the same specifications:
+The OpenAPI Schema Object is a JSON Schema vocabulary which extends JSON Schema Core and Validation vocabularies. As such any keyword available for those vocabularies is by definition available in OpenAPI, and will work the exact same way, including but not limited to:
```

Comment:
What is "**a** JSON Schema vocabulary"? is it a fork of the "JSON Schema vocabulary"? if so, shouldn't we call it the "OpenAPI vocabulary" as it is referred down below? 

Where will this vocabulary be located? Can we see it? It would be nice to link it here, especially if tooling is required to support it.

Reply:
A "JSON Schema vocabulary" is a specification which defines a set of keywords which have specific semantic meaning when applying a JSON Schema to a JSON instance.
A JSON Shema meta-schema defines the known vocabularies and if support for them is required in order to process the document.
You can see this in practice by looking at the General-purpose meta-schema for draft `2019-09` at http://json-schema.org/specification.html

The URI used as the vocabulary identifier, as like with $id, is not necessarily network addressable, and in the case of vocabularies, the identifier URI as a resources is reserved but not defined.

If you'd like to talk about this in more detail, feel free to come by the JSON Schema slack server, or reach out to any of the core team =]

Reply:
I understand the mechanics. I guess where I want to get with this is: how will this translate practically-speaking when implementing this. How will one be able to support OpenAPI Schemas using the Vocabulary mechanism if none is provided (in this repo)? Will this mean it will be up to the implementor to create its own Vocabulary file?

Reply:
@jstoiko I'm not sure why you think none will be provided.  This is about updating a certain part of the specification document to conform to `2019-09`.  It is not a PR to finalize every aspect of every issue related to updating JSON Schema ever.  There are various other issues and PRs filed.  PR #2016 in particular shows a first pass at a vocabulary meta-schema.

Please don't burden this PR with more work than it is already doing.  It is already doing a lot.

Reply:
> How will one be able to support OpenAPI Schemas using the vocabulary mechanism if none is provided (in this repo)? - @jstoiko 

The "vocabulary" IS the specification document which defines the additional keywords.

An implementer will implement a given vocabulary. There are implementations which DO stuff with OAS 3.0 flavour JSON Schema already, right?

An implementer will not "maintain a vocabulary". They may maintain a library which supports the vocabulary. As @handrews that, and a meta-schema for any potential vocabulary, is way beyond the scope of this PR.

Reply:
There’s no such thing as a vocabulary file so we’re good here.

Reply:
> There’s no such thing as a vocabulary file so we’re good here.

Correct.  #2016 shows one possible meta-schema for the vocabulary, with a vocabulary URI that I just made up.  The TSC may prefer to URI to either be a pure identifier, be the URI for the Schema Object specification, or be an identifier which redirects to the specification (which is more or less what we're doing with the official JSON Schema vocabulary URIs right now, although limitations of GitHub pages mean it's done in a slightly weird way using an HTLM refresh https://json-schema.org/draft/2019-09/vocab/applicator ).

The recommendation in the JSON Schema spec is that vocabulary URIs be pure identifiers, with the idea being that we can add a formal vocabulary file type later.  But that seems to confuse people too much so redirecting to the relevant specification is probably best for now.